### PR TITLE
add responsive layout

### PR DESCRIPTION
**Pull request by LiorRachi**
add responsive layout to login page

### CODE REVIEW

File: public/components/header.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import css from '../css/general.module.css'
```
Comment (alonpeled44) [inline public/components/header.jsx:1]: header css should be in a separate css file, each component should have it's own css file

File: public/components/header.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    const currentDate = new Date().toLocaleDateString('en-gb', { day: "numeric", month: "numeric", year: "numeric" });
+    return (
+        <header className={css.header}>
+            <div>
+                <img src="https://cdn-icons-png.flaticon.com/128/868/868596.png" alt="pokeball" />
+                <h1>Pokèmon</h1>
+            </div>
+            <div>
+                <h1>{currentDate}</h1>
+            </div>
```
Comment (alonpeled44) [inline public/components/header.jsx:13]: unneeded div

File: public/components/loginForm.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import css from '../css/login.module.css';
+export default function LoginForm() {
+    return (
+        <main className={css.mainWrapper}>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:4]: main should be ONLY in main (index). If you want to have a wrapper to the component/page add it as a div with a respective class/id.

File: public/components/loginForm.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            <div className={css.blurryBackground}></div>
+            <section className={css.main}>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:6]: cahnge name

File: public/components/loginForm.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+                <section className={css.formWrapper}>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:7]: not a section - change to div.

File: public/components/loginForm.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+                    <form className={css.form}>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:8]: no need for class

File: public/components/loginForm.jsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+                        <h1 className={css.title}>L<span><img src="https://cdn-icons-png.flaticon.com/128/868/868596.png" /></span>gin</h1>
+                        <section className={css.inputs}>
+                            <input className={css.input} type="text" id="_username" placeholder="username..." />
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:11]: no need for class

File: public/components/loginForm.jsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+                            <input className={css.input} type="password" id="_password" placeholder="password..." />
+                            <h6 className={css.msg}></h6>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:13]: not a header - it does not explain what this segment is about. change to `<p></p>`

File: public/components/loginForm.jsx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+                        </section>
+                        <section className={css.buttons}>
+                            <button className={css.button} id="login-btn" type="button">Login</button>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:16]: no need for class

File: public/css/general.module.css [hunk 10]
```diff
@@ -40,6 +44,40 @@
     gap: 10px;
 }
 
+.header div:last-child {
+    margin-right: 20px;
```
Comment (alonpeled44) [inline public/css/general.module.css:48]: no.

File: public/css/general.module.css [hunk 11]
```diff
@@ -40,6 +44,40 @@
...
+}
+
 .header h1 {
     font-size: 42px;
+}
+
+@media only screen and (max-width: 1200px) {
+    .header {
+        position: absolute;
```
Comment (alonpeled44) [inline public/css/general.module.css:57]: if it is the same in all queries no need to mention it again

File: public/css/general.module.css [hunk 12]
```diff
@@ -40,6 +44,40 @@
...
+        z-index: 2;
```
Comment (alonpeled44) [inline public/css/general.module.css:58]: why? z-index 1 should work

File: public/css/login.module.css [hunk 13]
```diff
@@ -88,7 +119,66 @@
     border-radius: 10px;
     background-color: transparent;
     color: #fff;
-    font-weight: 600d;
+    font-weight: 600;
```
Comment (alonpeled44) [inline public/css/login.module.css:122]: if the font family only have one face, the weight will always be consistent. no need for this.

**Review by LiorRachi — COMMENTED** [review]
i cant approve, i created the PR ;)

File: public/components/loginForm.jsx [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+            <div className={css.formWrapper}>
+                <form>
+                    <h1>L<span><img src="https://cdn-icons-png.flaticon.com/128/868/868596.png" /></span>gin</h1>
+                    <section className={css.inputs}>
+                        <input type="text" id="_username" placeholder="username..." />
+                        <input type="password" id="_password" placeholder="password..." />
+                        <p></p>
+                    </section>
+                    <section className={css.buttons}>
+                        <button className={css.button} id="login-btn" type="button">Login</button>
```
Comment (alonpeled44) [inline public/components/loginForm.jsx:15]: remove unneeded class

File: public/css/header.module.css [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+    margin: 0;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    position: absolute;
+    z-index: 0;
+}
+
+.header> {
```
Comment (alonpeled44) [inline public/css/header.module.css:14]: use &> in .header

**Review by alonpeled44 — APPROVED** [review]
16 comments total

**Pull request merged**